File: py101/easy3/4.py
# https://launchschool.com/exercises/b7b7220a

def stringy(num):
    string = []

    for i in range(num):
        if i % 2 == 0:
            string.append("1")
        else:
            string.append("0")
    return "".join(string)


print(stringy(6) == "101010")           # True
print(stringy(9) == "101010101")        # True
print(stringy(4) == "1010")             # True
print(stringy(7) == "1010101")          # True

File: js210/fundamentalExercises/med2/4.js
// https://launchschool.com/exercises/4b0d0727


function makeDoubler(caller) {
  return number => {
    console.log(`This function was called by ${caller}.`);
    return number + number;
  };
}


const doubler = makeDoubler('Victor');
let num = doubler(5);
console.log(num);